webdriver/tests/element_click/support/close_window.html

```diff
@@ -0,0 +1,9 @@
+<html>
+    <body onload="checkload()">
+        <script>
+        function checkload() {
+            window.close()
+        }
+        </script>
+    </body>
+</html>
```

Comment:
Make this inline.

Reply:
I don't quite understand how to do this. Right now I have a link inline that when is clicked on, it will go to this html file. I'm not sure how this could be inline with the href.

Reply:
You can give data URIs to links as well.

Reply:
So I tried using data URls but it doesn't work on chrome or edge. It only works in firefox. The test needs to pass two browsers for the spec to go to recommendation correct? Do you still want me to add data URls.

Reply:
At least for Chrome it should work fine. I can open `data:text/html,%3Ch1%3EHello%2C%20World!%3C%2Fh1%3E` without issues.

Reply:
That opened for me as well but didn't seem to open in an a tag element.
<a href=data:text/html,%3Ch1%3EHello%2C%20World!%3C%2Fh1%3E>a</a>

Reply:
Ok, now I see. This should actually not be allowed and as such cannot work. See https://blog.mozilla.org/security/2017/11/27/blocking-top-level-navigations-data-urls-firefox-58/ for details. So in case of navigation we cannot use data URLs and have to rely on HTML testcases served by wptserve.

Reply:
OK, seems fine to me then.